LogicalTypes.md

```diff
@@ -255,6 +255,16 @@ The primitive type is a 2-byte fixed length binary.
 
 The sort order for `FLOAT16` is signed (with special handling of NANs and signed zeros); it uses the same [logic](https://github.com/apache/parquet-format#sort-order) as `FLOAT` and `DOUBLE`.
 
+### FIXED_SIZE_LIST
+
+The `FIXED_SIZE_LIST` annotation represents a fixed-size list of elements
+of a primitive data type. It must annotate a `binary` primitive type.
```

Comment:
"binary" means either fixed or variable length, right? I always get confused 😅.

Reply:
Could you please provide a concrete example on how the list is structured? What about their definition & repetition levels? Intuitively, I thought not limit it to binary type. For example, it would be possible to support something like int[N] or double[N] and even multi-dimensional list like int[M][N].

Reply:
Perhaps use `byte_array` in this PR (see #251).

Reply:
Will do, thanks!

Reply:
> Could you please provide a concrete example on how the list is structured? What about their definition & repetition levels? Intuitively, I thought not limit it to binary type. For example, it would be possible to support something like int[N] or double[N] and even multi-dimensional list like int[M][N].

I would represent the fixed sized list as a non-nested `FIXED_LEN_BYTE_ARRAY` + `type` + `num_values`. Multidimensional lists/arrays bring much more complexity that I'm not sure makes sense to store as a logical type (see [FixedShapeTensor in Arrow](https://arrow.apache.org/docs/format/CanonicalExtensions.html#fixed-shape-tensor)).

> Perhaps use `byte_array` in this PR (see #251).

Done.